### DISCUSSION TITLE
I'm unable to extract zip installers to non C:/ drive

### DISCUSSION THREAD

**Discussion by csc530** [microsoft/winget-cli#3092] [Q&A] [answered]
I have tried to install several portable apps such as
- `ajeetdsouza.zoxide`
- `zyedidia.micro`
- `AntiMicroX.antimicroxPortable`
 which fail with an error of 
`An unexpected error occurred while executing the command:`
`copy_file: Access is denied.: "%Temp%/winget/PACKAGEID/extracted/PACKAGE.VERSION" "D:/Apps/winget/users/PACKAGE..."`
`Portable install failed; Cleaning up...`

and in the logs it says a lot of `rename: Access is denied` from the temp extracted folder to the destination drive and one `std::exception: create_hard_link: The system cannot move the file to a different disk drive.` before the error is caught in the next line in the log

In my winget settings my install behaviour is
```json
"defaultInstallRoot": "D:/Apps/WinGet/Packages",
"portablePackageMachineRoot": "D:/Apps/WinGet/Machine/",
"portablePackageUserRoot": "D:/Apps/WinGet/User/",
"preferences": {
    "architectures": [
        "x64"
    ],
    "locale": [
        "en-CA",
        "CA",
        "en",
        "fr-CA",
        "fr"
    ],
    "scope": "machine"
}
```

 but
- `Rclone.Rclone`
- `ahmetb.kubens`
- `wagoodman.dive`

are installed successfully which are also zip installers

I wasn't sure if this should be an issue or if perhaps I have a problem with my windows or winget configuration.

using `Windows Package Manager v1.4.10173`

**Comment by Trenly** [comment, marked as answer]
* Duplicate of #2931; Resolved by #3003

**Reply by csc530** [reply]
Thank you, I haven't seen this.
So I can expect a fix in the next update (since it's labelled with the v1.5 milestone?)

**Reply by Trenly** [reply]
Yes, it should be part of the next release
